_releases/2014-12-31-crystal-0.5.6-release.md

```diff
@@ -52,7 +52,8 @@ and people won't have to worry about which documentation generator to choose. We
 is more than just the language itself and that good integrated tools are indispensable for a good,
 smooth experience.
 
-Now there's just one thing missing: the documentation itself! We will add it, little by little, and you
-are very welcome to help us out. Luckily [Yukihiro Matsumoto kindly let us use some Ruby documentation as
-a starting point](https://twitter.com/yukihiro_matz/status/549317901002342400), given that a lot of
-APIs are very similar, if not equal. So どうもありがとうございます to him :-)
+> **THANKS:**
+> Now there's just one thing missing: the documentation itself! We will add it, little by little, and you
+> are very welcome to help us out. Luckily [Yukihiro Matsumoto kindly let us use some Ruby documentation as
+> a starting point](https://twitter.com/yukihiro_matz/status/549317901002342400), given that a lot of
+> APIs are very similar, if not equal. So どうもありがとうございます to him :-)
```

Comment:
I don't think this callout makes sense here. This is specific to the documentation and a call to action more than an actual thanking.

```suggestion
Now there's just one thing missing: the documentation itself! We will add it, little by little, and you
are very welcome to help us out. Luckily [Yukihiro Matsumoto kindly let us use some Ruby documentation as
a starting point](https://twitter.com/yukihiro_matz/status/549317901002342400), given that a lot of
APIs are very similar, if not equal. So どうもありがとうございます to him :-)
```

Reply:
Good catch! 😅